packages/siop-oid4vp/lib/rp/Opts.ts

```diff
@@ -55,6 +60,7 @@ export const createVerifyResponseOptsFromBuilderOrExistingOpts = (opts: {
   return opts.builder
     ? {
         hasher: opts.builder.hasher ?? defaultHasher,
+//        correlationId: uuidv4(), We can't set a correlationId here, it will no longer check functions like  this.sessionManager.getCorrelationIdByNonce(resNonce, false)
```

Comment:
Yes, why would it need that function to begin with if you already have a correlation Id? and internally we work by correlationId

Reply:
Guess we'll never find out anymore...
![image](https://github.com/user-attachments/assets/443e7cc7-864e-4512-b212-e02133795a4a)

I left the comment there in case anyone else would decide to "fix" something in the future